docs/v0.7/en/user-guide/operations/region-migration.md

```diff
@@ -37,15 +39,24 @@ The region migration won't be performed if the `from_peer_id` equals the `to_pee
 
 Remember, if you deploy the cluster via the GreptimeDB operator, the `peer_id` of Datanode always starts from 0. For example, if you have a 3 Datanode GreptimeDB cluster, the available `peer_id` will be 0,1,2.
 
-Finally, you can do a Region migration request via the following HTTP request:
+Finally, you can do a Region migration request via the following SQL:
 
 ```bash
-curl "{YOUR_META_SRV_LEADER_SERVER}:3002/admin/region-migration?region_id={REGION_ID}&from_peer_id={FROM_PEER_ID}&to_peer_id={TO_PEER_ID}&replay_timeout=5m"
+select migrate_region(4398046511104, 1, 2, 60);
+```
+
+The parameters of `migrate_region`：
+
+```sql
+select migrate_region(region_id, from_peer_id, to_peer_id, replay_timeout);
 ```
 
 | Option           | Description                                                    | Required     |   |
 |------------------|----------------------------------------------------------------|--------------|---|
 | `region_id`      | The region id.                                                 | **Required** |   |
 | `from_peer_id`   | The peer id of the migration source(Datanode).                 | **Required** |   |
 | `to_peer_id`     | The peer id of the migration destination(Datanode).            | **Required** |   |
-| `replay_timeout` | The timeout of replay data. e.g., 1h30m10s                     | **Required** |   |
\ No newline at end of file
+| `replay_timeout` | The timeout(secs) of replay data.                              |   Optional   |   |
+
+Note: If the new Region fails to replay the data within the specified timeout, 
```

Comment:
Do we have some Note abomination like:

<img width="885" alt="image" src="https://github.com/GreptimeTeam/docs/assets/18818196/d49b6959-a690-4150-a754-b472026badf5">

cc @alili 

This is not a blocker for this PR, though.

Reply:
Seems:

````
:::note

...

:::
````
?

Reply:
let me fix this